modules/nw-ovn-kuberentes-limitations.adoc

```diff
@@ -11,6 +11,11 @@ The OVN-Kubernetes Container Network Interface (CNI) cluster network provider ha
 The default value, `cluster`, is supported for both parameters.
 This limitation can affect you when you add a service of type `LoadBalancer`, `NodePort`, or add a service with an external IP.
 
+[NOTE]
+====
+For openshift deployments in cloud platforms except IBM cloud and Power VS, the ingress operator create Loadbalancer type service for the ingress with `ExternalTrafficPolicy` pointing to local, however the load is still distributed to the available backends until the fix is available, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1903408[BZ#1903408].
```

Comment:
replace **openshift** with **{product-title}** 
in **all** cloud platforms
Capitalize **Ingress** and **Operator**
Replace **create** with **creates**
Add an article **a** before `Loadbalancer type service for....`

Reply:
Please squash all the five commits into one commit as per OpenShift doc guidelines.

Reply:
Hi @Amrita42 

We might not need this anymore as the engineering has agreed to change product behavior which the doc bug was going to cover. I'll close this if not needed in a day or two.

Reply:
yeah sure , when closing the bug , you can mention as not a bug. Thanks.